pkg/block/fetcher.go

```diff
@@ -463,6 +480,12 @@ func (f *DeduplicateFilter) Filter(metas map[ulid.ULID]*metadata.Meta, synced Ga
 	}
 }
 
+// GetDuplicateIDs returns slice of block ids
+// that are filtered out by DeduplicateFilter.
+func (f *DeduplicateFilter) GetDuplicateIDs() []ulid.ULID {
```

Comment:
Get/Set prefixes for Go language are not preferred, so just 
```suggestion
func (f *DeduplicateFilter) DuplicateIDs() []ulid.ULID {
```
...is totally fine.